tests/openvino/test_modeling.py

```diff
@@ -619,6 +623,27 @@ def test_pipeline(self, model_arch):
         self.assertTrue(isinstance(outputs[0]["label"], str))
         gc.collect()
 
+    @parameterized.expand(TIMM_MODELS)
```

Comment:
I think we should have a test which:
- load a timm model from hub
- saves it in IR locally (.save_pretrained)
- loads IR and does inference (random data is ok)

Reply:
A test `test_timm_save_and_infer` is added.